docs/source/app_developers_guide/about_event_subscriptions.rst

```diff
@@ -0,0 +1,137 @@
+**************************
+About Events Subscriptions
+**************************
+
+An event subscription consists of an event type (the name of the event) and an
+optional list of filters for this event type.
+
+An event is “in a subscription” if the event’s ``event_type`` field matches the
+subscription’s ``event_type`` field, and the event matches any filters that are
+in the event subscription. If multiple filters are included in a subscription,
+only events that pass *all* filters will be received.
```

Comment:
Suggest adding that "However, you may have multiple subscriptions at a time, and will receive all events in any of these subscriptions."